### PR TITLE
Configuring T-Shirt sizing for Redis and OS

### PR DESCRIPTION
**Pull request by shehzadashiq**
Defining sizing according to current CF/PaaS specifications

### CODE REVIEW

**Review by ejayesh — COMMENTED** [review]
why has redis small been changed from small to medium and redis medium to large?

**Comment by shehzadashiq** [comment]
> why has redis small been changed from small to medium and redis medium to large?

I looked at what we had in CF/PaaS and then matched accordingly. For instance PaaS defines small as 

3GB RAM, single node, no failover

This matches 'cache.t4g.medium' which I then assigned to the small instance.

**Review by lgarvey — APPROVED** [review]
Looking good! 🚀

File: storage-plans.yaml [hunk 1]
```diff
@@ -1,50 +1,129 @@
...
+
+  # 12.93GB RAM, single node, no failover.
+  large:
+    replicas: 0
+    instance: 'cache.m6g.large'
+
+  # 12.93GB RAM, highly-available
+  large-ha:
+    replicas: 2
+    instance: 'cache.m6g.large'
```
Comment (yusufsheiqh) [inline storage-plans.yaml:52]: These are `large`, so the instance should be larger by a factor of 1. They are currently the same as `medium`.

```suggestion
  large:
    replicas: 0
    instance: 'cache.m6g.xlarge'

  # 12.93GB RAM, highly-available
  large-ha:
    replicas: 2
    instance: 'cache.m6g.xlarge'
```

**Review by lgarvey — APPROVED** [review]
LGTM

File: storage-plans.yaml [hunk 2]
```diff
@@ -1,52 +1,188 @@
...
+    replicas: 2
+    instance: db.m5.large
+
+  # 564GB Storage, Postgres Version 13. DB Instance Class: db.m5.2xlarge
+  large-13:
+    volumesize: 564
+    replicas: 0
+    instance: db.m5.2xlarge
+  
+  # 564GB Storage, Postgres Version 13. DB Instance Class: db.m5.large
```
Comment (yusufsheiqh) [inline storage-plans.yaml:166]:
```suggestion
  # 564GB Storage, Postgres Version 13. DB Instance Class: db.m5.2xlarge
```

File: storage-plans.yaml [hunk 3]
```diff
@@ -1,52 +1,188 @@
...
+  xlarge-13:
+    volumesize: 2100
+    replicas: 0
+    instance: db.m5.4xlarge
+  
+  # 2TB Storage, Postgres Version 13. DB Instance Class: db.m5.4xlarge
+  xlarge-13-ha:
+    volumesize: 2100
+    replicas: 2
+    instance: db.m5.4xlarge
```
Comment (yusufsheiqh) [inline storage-plans.yaml:182]: I could be wrong here, but was the `2100` intentional?

```suggestion
  # 2TB Storage, Postgres Version 13. DB Instance Class: db.m5.4xlarge
  xlarge-13:
    volumesize: 2000
    replicas: 0
    instance: db.m5.4xlarge
  
  # 2TB Storage, Postgres Version 13. DB Instance Class: db.m5.4xlarge
  xlarge-13-ha:
    volumesize: 2000
    replicas: 2
    instance: db.m5.4xlarge
```

**Pull request merged**